iml-api/src/main.rs

```diff
@@ -36,6 +36,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     };
 
     let pool = iml_rabbit::connect_to_rabbit(2);
+    let rabbit_connection = pool.get().await?;
```

Comment:
I think we should pass in the pool rather than a single connection. If the connection dies the API will be unusable as well.

Reply:
Done